src/TLuaInterpreter.cpp

```diff
@@ -11777,13 +11793,81 @@ int TLuaInterpreter::getRoomChar(lua_State* L)
     if (!pR) {
         lua_pushnil(L);
         lua_pushfstring(L, "room with id %d does not exist", id);
-        return 2;
+        return 1;
     } else {
         lua_pushstring(L, pR->mSymbol.toUtf8().constData());
         return 1;
     }
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setRoomNameOffset
+int TLuaInterpreter::setRoomNameOffset(lua_State* L)
+{
+    int id;
+    qreal x, y;
+    if (!lua_isnumber(L, 1)) {
+        lua_pushfstring(L, "setRoomNameOffset: bad argument #1 type (room id as number expected, got %s!)",
+                       luaL_typename(L, 1));
+        return lua_error(L);
+    } else {
```

Comment:
And the same thing in other cases further down...

Reply:
I do know that that would be cleaner. However, for all of this I simply copied the existing code. Coding style diffs are all well and good but shouldn't get mixed with bug fixes or feature changes.

I can certainly go through that code and clean up all these return-else antipatterns, but I'll do that when everything else has settled down, otherwise there'll be unnnecessary merge and/or rebase conflicts.